### ISSUE TITLE
sort based on lastUpdated

### ISSUE THREAD

**Issue by fatemeh91** [awslabs/fhir-works-on-aws-deployment#212]
Hi, 
I have 3 entries with same name (ABC) and i want to sort them based on the lastUpdated and get the last one so i did:

String pt_url2 = " https://uwiv2v70ia.execute-api.us-west-2.amazonaws.com/dev/Patient?name=ABC&_sort=-lastUpdated&_count=1";
		URL url3 = new URL(pt_url2);//.openConnection();
		HttpURLConnection connection2 = (HttpURLConnection) url3.openConnection();
		connection2.setRequestProperty("Content-Type", "application/json; charset=utf-8");
		connection2.setRequestProperty("Authorization", cognitoAuthToken);
		connection2.setRequestProperty("x-api-key", API_KEY);
		connection2.setRequestMethod("GET");
		InputStream response2 = connection2.getInputStream();
		try (Scanner scanner = new Scanner(response2)) {
		    String responseBody = scanner.useDelimiter("\\A").next();
		    System.out.println(responseBody);
		    
		}

but it returns me an empty entry. if i remove the _sort and _count then I will get all my three entries.

**Comment by nguyen102** [comment]
Hi,

We currently don't support the `_sort` parameter. Using `_sort` should return a 400 error with the body 
```
{
    "resourceType": "OperationOutcome",
    "text": {
        "status": "generated",
        "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><h1>Operation Outcome</h1><table border=\"0\"><tr><td style=\"font-weight: bold;\">error</td><td>[]</td><td><pre>Invalid search parameter '_sort' for resource type Patient</pre></td></tr></table></div>"
    },
    "issue": [
        {
            "severity": "error",
            "code": "invalid",
            "diagnostics": "Invalid search parameter '_sort' for resource type Patient"
        }
    ]
}
```

Can you please confirm what version of FWoA you're running? You can check that by looking at the version number in "package.json".

The latest FWoA version is `2.4.0` with the commit sha `72f4fa23feb68456a924a013dd064beaf75f1b41`

**Comment by rmharrison** [comment]
@nguyen102 I foresee many of these "Does AWS FHIR Works support <x>?" questions. In anticipation, would you consider?

- good: Publishing a static CapabilityStatement with your release note [^1]
- better: Running a live reference implementation, so folks can hit the `metadata` and `.well-known/smart-configuration` endpoints themselves
  * Optional: Add yourself to the HL7 Confluence of test/reference servers, [Public Test Servers](https://confluence.hl7.org/display/FHIR/Public+Test+Servers) and/or [Test Servers](https://confluence.hl7.org/display/FHIR/Test+Servers) 
  * Optional: Publishing your reference server conformance with AEGIS Touchstone

[^1] It's awesome you added the Postman collection, but this requires running the application locally.
> To know what all this FHIR API supports please use the GET Metadata postman to generate a Capability Statement.
> https://github.com/awslabs/fhir-works-on-aws-deployment/blob/72f4fa23feb68456a924a013dd064beaf75f1b41/README.md#using-postman-to-make-api-requests

**Comment by nguyen102** [comment]
Thanks for the suggestion Harrison. We have discussed about hosting a public FHIR server, but have decided to punt that for now, because of the legal and security approvals required to host one. It's always great to hear from customers, and I will relay this feature request back to the team. 

Maybe in the short term we can publish a static CapabilityStatement like you suggested.

**Comment by rsmayda** [comment]
The original issue of _sort not working on dates has been resolved: https://github.com/awslabs/fhir-works-on-aws-search-es/issues/60 and released in [2.5.0](https://github.com/awslabs/fhir-works-on-aws-search-es/blob/mainline/CHANGELOG.md#250-2021-04-23)
